third_party/move/move-prover/tests/sources/functional/choice.v2_exp

```diff
@@ -11,7 +11,6 @@ error: post-condition does not hold
    =         return = <redacted>
    =     at tests/sources/functional/choice.move:17: simple_incorrect
    =         result = <redacted>
```

Comment:
Why would we see changes in the prover output? CFG simplification would be turned off here (because it is off by default), right?

Reply:
Forgot to update the prover tests after turning off CFG simplification. Done.